.vscode/tasks.json

```diff
@@ -0,0 +1,89 @@
+{
+    "version": "2.0.0",
+    "presentation": {
+      "echo": false,
+      "reveal": "always",
+      "focus": false,
+      "panel": "shared",
+      "showReuseMessage": false,
+      "close": false
+    },
+    "tasks": [
+      {
+        "label": "Create terminals",
+        "dependsOrder": "sequence",
+        "dependsOn": [
+          "Echo command",
+          "Install dependencies",
+          "Open README",
+          "Echo ready command",
+          "Open Terminal",
+        ],
+        // Mark as the default build task so cmd/ctrl+shift+b will create them
```

Comment:
is this valid to have the `//` here?

Reply:
ah, neato: https://code.visualstudio.com/docs/languages/json#_json-with-comments